lib/gooddata/models/process.rb

```diff
@@ -186,14 +186,18 @@ def deploy_component(data, options = { client: GoodData.client, project: GoodDat
         client, project = GoodData.get_client_and_project(options)
         data = { process: data } unless data[:process]
         data[:process] = GoodData::Helpers.symbolize_keys(data[:process]).select { |k| %i[type name component].include? k }
-        data[:process][:component] = GoodData::Helpers.symbolize_keys(data[:process][:component]).select { |k| %i[name version configLocation].include? k }
-
-        # a feature flag is required to create component type processes
-        client.post "/gdc/projects/#{project.pid}/projectFeatureFlags",
-                    featureFlag: {
-                      key: 'enableEtlComponent',
-                      value: true
-                    }
+        data[:process][:component] = GoodData::Helpers.symbolize_keys(data[:process][:component]).select { |k| %i[name version configLocation config].include? k }
+
+        # a feature flag is required to create ADDv2 components
+        if data[:process][:component][:name] == 'gdc-data-distribution'
+          client.post(
+            "/gdc/projects/#{project.pid}/projectFeatureFlags",
+            featureFlag: {
+              key: 'enableDataDistribution',
```

Comment:
enableDataDistribution replaces enableEtlComponent, or the second one is discontinued ?

Reply:
it is another one, no override each other.

Reply:
confused even more:
- before change, the elt-component wae enabled with every "deploy component" call
- now, only "data distribution" will be enabled, if and only if we process ADDv2 processes, in other words "etl component" will be never enabled, ... is this right ?

what is the purpose of these flags / what is the server behaviour when enabled  ?

Reply:
Currently, the FF enableEtlComponent and enableDataDistribution are true as default. So we can remove it, no need to set FF enableDataDistribution as well.